### ISSUE TITLE
Fork this Project for further Development

### ISSUE THREAD

**Issue by cedricbambooza** [corna/me_cleaner#365]
Hey, I've seen the many forks that are already there. But they are fairly old.

I'm wondering, is a motivated Maintainer/Developer out there, who has the Time and Ressources to fork this Project, so that it can be further developed? 

I'm not very sure why this hasn't happenend already, maybe there's a good reason. 

But I hope there is not.

**Comment by FrostKnight** [comment]
Yes, its well overdue, only thing is, who has the skill, intelligence and time to do so?

Those are things to keep in mind for whoever decides to do so.

**Comment by cedricbambooza** [comment]
@FrostKnight  Yes you are right. 

One can see that (far too) many have tried or not but still have forked the project. 

But none seem to have succeeded. 

Me personally have some skill with code and maybe also the time to be there regularly for the project. 

But to be sure that every new Code Patch does what it has to and the documentation stays up to date, I may need some help from skilled people. 

I could not edit the code up to supporting newest Intel Chipsets on my own. 

I simply don't have the knowledge of the Management Engine. :)

**Comment by FrostKnight** [comment]
I on the other hand, know almost nothing about its functionality besides what is already known.  Technology wise, I don't know how to even contribute patches to this, let alone maintain a fork.  I am but a humble spectator/supporter.
I would've thought someone would know how to do this who has time.  
-_-

Edit: I wonder if people won't due to fear? Hmm, I guess maybe? Idk...
